### PR TITLE
release 2.1.0-snapshot.20220308.9472.0.1f24348e

### PR DESCRIPTION
**Pull request by stewartstewart-da**
@stewartstewart-da is in charge of this release

### Pull Request Checklist

- [X] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [X] Include appropriate tests
- [X] Set a descriptive title and thorough description
- [X] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [X] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: LATEST [hunk 1]
```diff
@@ -1,3 +1,4 @@
+1f24348e2178fb2dab022ff0cf3d75c5dfa484a6 2.1.0-snapshot.20220308.9472.0.1f24348e
```
Comment (cocreature) [inline LATEST:1]: this should be a split release if at all but as before, you don’t actually need this. the nightly release is good enough.

**Comment by stewartstewart-da** [comment]
Manual test passed on Windows

**Comment by stewartstewart-da** [comment]
Manual test passed on macOS

File: LATEST (hunk 1 above)
Comment (stewartstewart-da) [reply LATEST:1]: @cocreature I thought the nightly release was the output of `deps list`? Am I supposed to get them from somewhere else?

Comment (S11001001) [reply LATEST]: `deps list` lists nightlies that have already been completed, and are therefore eligible for complete split release.

**Pull request closed without merging**